opensrp-reveal/src/main/assets/json.form/bednet-distribution.json

```diff
@@ -0,0 +1,351 @@
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        }
+      },
+      {
+        "key": "numBedSpaces",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "normal_edit_text",
+        "hint": "Number of bed spaces *",
+        "edit_type": "number",
+        "is_mandatory": true,
+        "v_required": {
+          "value": true,
+          "err": "The number of bed spaces is required"
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        }
+      },
+      {
+        "key": "calcNumNetsNeeded",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "hidden",
+        "calculation": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "bednet-distribution-calculation-rules.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "displayRecommendedNumNets",
+        "type": "label",
+        "label_text_style": "bold",
+        "text": "{calcNetsNeeded} nets are recommended",
+        "text_color": "#000000",
+        "v_required": {
+          "value": true
+        }
+      },
+      {
+        "key": "existingNets",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "normal_edit_text",
+        "hint": "Total number of existing Nets *",
+        "edit_type": "number",
+        "value": "0",
+        "is_mandatory": true,
+        "v_required": {
+          "value": true,
+          "err": "The total number of existing nets is required"
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        }
+      },
+      {
+        "key": "existingLLINs",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "normal_edit_text",
+        "hint": "How many of these nets are LLINs that are in good condition? *",
+        "edit_type": "number",
+        "value": "0",
+        "is_mandatory": true,
+        "v_required": {
+          "value": true,
+          "err": "The total number of existing LLINs is required"
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        },
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "bednet-distribution-relevance-rules.yml"
```

Comment:
I haven't created relevance rules here. It should be (existingLLINs + existingITNs)==existingNets 
To put it in plain language, the total number of existing LLINs and ITNS needs to be = to the number entered in existingNets. If that's not the case, we need to add an error message until it's satisfied.

Reply:
Use cumulative validator on either existingLLINs and/or existingITNs
 "v_cumulative_total": {
          "value":"existingNets",
          "related_fields": [
            <<other related field>>
          ],
          "err": <<Error message to be displayed>>
        }